metron-analytics/metron-profiler-client/src/main/java/Window.tokens

```diff
@@ -0,0 +1,17 @@
+COMMA=1
```

Comment:
I haven't found a way.  I looked into it with Stellar and it didn't seem possible at the time.